docs/authentication/configuration/session-options.mdx

```diff
@@ -21,52 +21,45 @@ Fortunately, with Clerk you have to ability to fully control the lifetime of you
 
 ### Inactivity timeout
 
-Denotes the duration after which a session will expire and the user will have to sign in again, if they haven't been active on your site. By default, this setting is disabled for all newly created instances. To enable it and set your desired value, in the Clerk Dashboard, go to **[Sessions](https://dashboard.clerk.com/last-active?path=sessions)**.
+Denotes the duration after which a session will expire and the user will have to sign in again, if they haven't been active on your site. A user is considered inactive when the application is closed, or when the app stops refreshing the token.
 
-<Images
-width={3024}
-height={1652}
-src="/docs/images/authentication/session-options/inactivity-timeout.jpg"
-alt="The 'Sessions' configuration section in the Clerk Dashboard with a red outline around the 'Inactivity timeout' section. This option is toggled on, revealing an input for customizing the time duration until inactivity timeout. A red arrow is pointing to the 'Apply Changes' button."
-/>
+By default, this setting is disabled for all newly created instances. To enable it and set your desired value:
+
+1. Navigate to the Clerk Dashboard.
+2. Select your application, then select **[Sessions](https://dashboard.clerk.com/last-active?path=sessions)** from the navigation sidebar.
+3. Toggle the **Inactivity timeout** switch to enable it.
+4. Set your desired duration.
 
 ### Maximum lifetime
 
-Denotes the duration after which a session will expire and the user will have to sign in again, regardless of their activity on your site. By default, this setting is enabled with a default value of 7 days for all newly created instances. To find this setting and change the value, in the Clerk Dashboard, go to **[Sessions](https://dashboard.clerk.com/last-active?path=sessions)**.
+Denotes the duration after which a session will expire and the user will have to sign in again, regardless of their activity on your site.
+
+By default, this setting is enabled with a default value of 7 days for all newly created instances. To find this setting and change the value:
 
-<Images
-width={3024}
-height={1652}
-src="/docs/images/authentication/session-options/maximum-lifetime.jpg"
-alt="The 'Sessions' settings in the Clerk Dashboard with a red outline around the 'Maximum lifetime' section. This option is toggled on, revealing an input for customizing duration of a session lifetime."
-/>
+1. Navigate to the the Clerk Dashboard.
```

Comment:
Same suggestions for consolidating these steps as above:
https://github.com/clerk/clerk-docs/pull/788/files#r1518068995